Dockerfile

```diff
@@ -115,7 +115,8 @@ RUN appium driver install uiautomator2 && \
 # Custom mcloud patches
 COPY files/mcloud/ /opt/mcloud
 # do not make backups because unpatched js files in the same folder might be used by Appium
-RUN cp -r -v /opt/mcloud/* ${APPIUM_HOME}
+RUN cp -r -v /opt/mcloud/* ${APPIUM_HOME} && \
+    npm install --prefix ${APPIUM_HOME}
```

Comment:
no sense to add extra install in such scenario.
we should adjust patch file having only diffs or cloning our own forked appium repository with changes, i.e. `cp -r -v` should be removed